ronn/geopm_agent_fixed_frequency.7.ronn

```diff
@@ -0,0 +1,137 @@
+The fixed frequency agent adjusts the CPU, Uncore, and Accelerator frequencies
+based upon the agent policy.  The Agent policy provides a single frequency value
+for all three frequency domains.  The uncore policy value is used to set both
+the minimum and maximum.  If the policy value for the CPU or Uncore is NAN,
+the frequency of that domain is not modified.  If the policy value for the
+Accelerator is NAN, the maximum Accelerator frequency is used.
+
+## AGENT NAME
+The agent described in this manual is selected in many geopm
+interfaces with the "fixed_frequency" agent name.  This name can be
+passed to **geopmlaunch(1)** as the argument to the `--geopm-agent`
+option, or the `GEOPM_AGENT` environment variable can be set to this
+name (see **geopm(7)**).  This name can also be passed to the
+**geopmagent(1)** as the argument to the '-a' option.
+
+## POLICY PARAMETERS
+
+  `ACCELERATOR_FREQUENCY`: The operating frequency in units of Hz for the
+                           Board Accelerators.  Setting it to a NAN will
+                           result in the system default value.
+  `CPU_FREQUENCY`: The operating frequency in units of Hz for all CPUs.
+                   Setting it to a NAN will result in the system default
+                   value.
+  `UNCORE_FREQUENCY`: The operating frequency in units of Hz for the uncore
+                      domain.  If specified the uncore clock will operate
+                      the fixed frequency provided.  If the parameter is
+                      NAN, then the system default range of uncore frequency
+                      will be allowed.
+
+## POLICY REQUIREMENTS
+The fixed frequency values must be in the order of ACCELERATOR_FREQUENCY,
+CPU_FREQUENCY, UNCORE_FREQUENCY.  A value must be provided, but can be NAN.
+
+## REPORT EXTENSIONS
+None
+
+## CONTROL LOOP RATE
+The agent gates the control loop to sample the region hash and control
+CPU frequency at 50 millisecond intervals.  If the uncore frequency is
+specified in the policy, then it is set at agent start up time.
+
+
+## EXAMPLES
+The recommended way to generate a policy file for this agent is to use the
+**geopmagent(1)** command line tool.
+
+To create a policy with an accelerator frequency of 1.53 GHz, a core
+frequency of 3.1 GHz, and an uncore frequency of 2 GHz
+
+    geopmagent -a fixed_frequency -p '1.530e9,3.1e9,2e9'
+
+Although the **geopmagent(1)** is the recommended tool for creating
+the json policy string, the json string without this tool can be used.
+For example, the above policy json is:
+
+    {"ACCELERATOR_FREQUENCY": 1530000000,
+     "CORE_FREQUENCY": 3100000000,
+     "UNCORE_FREQUENCY": 2000000000}
+
+The **geopmread(1)** command line tool can be useful for learning the
+bounds of these system parameters.
+
+The minimum and maximum accelerator frequencies are queried as below:
+
+    $ geopmread NVML::FREQUENCY_MIN board 0
+    135000000
+
+    $ geopmread NVML::FREQUENCY_MAX board 0
+    1530000000
+
```

Comment:
Check for name changes GPU_FREQUENCY_MIN_LIMIT etc (Brad's PR)
Update code and documentation

Reply:
also include #1945 and then change this...

Reply:
Pulled in latest dev which uses aliases from #1945